### ISSUE TITLE
[Article] Instant SSO for Django

### ISSUE THREAD

**Issue by xgp** [p2-inc/phasetwo-docs#56]

**Comment by xgp** [comment]
- https://blog.jonharrington.org/static/integrate-django-with-keycloak
- https://www.informaticsmatters.com/blog/2021/06/01/installing-keycloak-on-Django-rest-framework.html
- https://github.com/Amsterdam/keycloak_oidc
- https://mozilla-django-oidc.readthedocs.io/en/stable/drf.html
- https://pypi.org/project/mozilla-django-oidc/

**Comment by pnzrr** [comment]
Additional Frameworks
- NextJs
- NuxtJs
- React
- VueJs
- Express
- Remix
- Astro

**Comment by xgp** [comment]
I think there are two broad categories here. There are frameworks that have a base idea of a user/role model (e.g. Django, Laravel, Rails(?), etc.), and then frameworks that are just expressing an opinion of how to build an app, without any high-level objects (e.g. Next, Vue, React, etc.). In the first case, the articles/docs should be focused on how to replace/augment the existing user management system with Keycloak and Phase Two SSO. In the second case, the articles/docs should be more focused on how to secure a page.

**Comment by pnzrr** [comment]
How to secure a page is the perfect starter use case.

**Comment by xgp** [comment]
Agreed. I think having a repo where we have examples for all frameworks that could be referenced from articles/docs/forums would be great. I'm surprised/unsurprised it doesn't exist already for Keycloak.
